api/clients/v2/relay_payload_retriever.go

```diff
@@ -130,7 +118,9 @@ func (pr *RelayPayloadRetriever) GetPayload(
 	for _, val := range indices {
 		relayKey := relayKeys[val]
 
-		blob, err := pr.getBlobWithTimeout(ctx, relayKey, blobKey)
+		blobLengthSymbols := eigenDACert.BlobInclusionInfo.BlobCertificate.BlobHeader.Commitment.Length
+
+		blob, err := pr.getBlobWithTimeout(ctx, relayKey, blobKey, blobLengthSymbols)
```

Comment:
here's another reason why I don't like `get`ter prefixes. Because then its unclear that this `getBlobWithTimeout` here is fetching over the network (the ctx gives it away but still).

would `retrieveBlobWithTimeout` make more sense?